src/components/CancelButton.tsx

```diff
@@ -1,20 +1,19 @@
-import { Chip } from '@material-ui/core';
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import { Chip } from '@mui/material';
```

Comment:
I think we should delete this component and just replace it with the new component using a cancel label

Reply:
I don't want this PR to generate a ton of new work. Scope of this PR is to carry over stuff from terraware-web and use them there.
We can clean this up later.
It would be great if we can create a new Jira to capture all the stuff that needs to get cleaned up?

Reply:
Note: ask Chudo for zeroheight and create a JIRA for cancel label.